Add documentation for log type updates in Security Analytics - pt. 2 (#3083)
opensearch-project/documentation-website: _security-analytics/usage/findings.md

# Working with findings

The **Findings** window includes features for viewing and working with findings. The two main features are:
* The bar graph with findings information arranged by count, date, and log type or rule severity.
* The **Findings** list arranged by time, finding ID, rule name, and other details.

> You can select the **Refresh** button at any time to refresh information on the **Findings** page.

Reviewer: We are now referring to button clicks with the word "Choose" as opposed to "Select" (global comment)

Author: We should as a team have a conversation about this. I have changed "select" to "choose" and "choose" to "select" in a number of PRs lately. Different reviewers, different interpretations of the style guide.

<img src="{{site.url}}{{site.baseurl}}/images/Security/date-pick.png" alt="Altering date range" width="55%">

As an alternative, you can select an option in the **Commonly used** section (see the preceding image of the calendar dropdown list) to conveniently set a window of time. Options include date ranges such as **Today**, **Yesterday**, **this week**, and **week to date**. 

When one of the commonly used windows of time is selected, you can choose **Show dates** in the date range field to populate the range of dates. Following that, you can select either the start date or end date to specify by an absolute, relative, or current date and time setting. For absolute and relative changes, choose **Update** to apply the changes.

As one more alternative, you can select an option from the **Recently used date ranges** section to go back to a previous setting.

## The Findings list

The **Findings** list displays all findings according to time of the finding, the finding ID, the rule name that generated the finding, the detector that captured the finding, and other details.

> <img src="{{site.url}}{{site.baseurl}}/images/Security/finding-list.png" alt="A list of all findings" width="85%">

Reviewer 2: Ensure that this image is introduced in the text surrounding it.

* The bell icon allows you to open the **Create detector alert trigger** pane, where you can quickly set up an alert for the specific finding and modify rules and their conditions as required.
For details on setting up an alert, see [Set up alerts]({{site.url}}{{site.baseurl}}/security-analytics/sec-analytics-config/detectors-config/#step-3-set-up-alerts) in detector creation documentation.

Each finding in the list also includes a **Finding ID**. In addition to using the diagonal arrow in **Actions**, you can select the ID to open the **Finding details** pane. An example of **Finding details** is shown in the following image.

<img src="{{site.url}}{{site.baseurl}}/images/Security/findings1.png" alt="Finding details pane" width="60%">

### Viewing surrounding documents

The **Finding details** pane contains specific information about the finding, including the document that generated the finding. To investigate the series of events that led to the finding or followed the finding, you can select **View surrounding documents** to open the document in the **Discover** panel and view other documents preceding or following it.

> 1. Open **Finding details** by selecting the **Finding ID** in the list of findings.

Reviewer 2: The list needs to be introduced by a brief sentence ending in a colon. What are we providing in the list?

Author: I'll change this to:
"Open **Finding details** by selecting the **Finding ID** in the **Findings** list."
The **Findings** list is talked about extensively in the parent heading preceding this level three heading.

1. In the **Documents** section, select **View surrounding documents**. If an index pattern already exists for the document, the **Discover** panel opens and displays the document. If an index pattern does not exist, the **Create index pattern to view documents** window opens and prompts you to create an index pattern, as shown in the following image.

    <img src="{{site.url}}{{site.baseurl}}/images/Security/findings2.png" alt="popup window prompting users to create an index pattern" width="60%">

> 1. In the **Create index pattern to view documents** window, the index pattern name is automatically populated. Enter the appropriate time field from the log index used to determine the timing for log events. For information on mapping log fields to detector fields, see [Step 2. Create field mappings]({{site.url}}{{site.baseurl}}/security-analytics/sec-analytics-config/detectors-config/#step-2-create-field-mappings). Select **Create index pattern**. The **Create index pattern to view documents** confirmation window opens.

Reviewer 2: ```suggestion
1. In the **Create index pattern to view documents** window, the index pattern name is automatically populated. Enter the appropriate time field from the log index used to determine the timing for log events. For information on mapping log fields to detector fields, see [Step 2. Create field mappings]({{site.url}}{{site.baseurl}}/security-analytics/sec-analytics-config/detectors-config/#step-2-create-field-mappings). Choose **Create index pattern**. The **Create index pattern to view documents** confirmation window opens.
```

## The Findings list

The **Findings** list displays all findings according to time of the finding, the finding ID, the rule name that generated the finding, the detector that captured the finding, and other details.

> <br><img src="{{site.url}}{{site.baseurl}}/images/Security/finding-list.png" alt="A list of all findings" width="85%">

Reviewer: The text following this image is displayed on the same line as the image because there is no new line between them. But this is existing documentation, so I would address it in another PR. Simply remove the `<br>` and put the image on line 42, and insert a new line after the image, so the "Use the..." text starts at line 44.

Author: Agreed. Will need to clean these up in another pass.

Use the **Rule severity** dropdown list to filter the list of findings by severity. Use the **log type** dropdown list to filter the list by log type.

The **Actions** column includes two options for each finding:

> * The diagonal arrow provides a way to open the **Finding details** pane, which describes the finding by parameters defined when creating the detector and includes the document that generated the finding.

Reviewer: Is it always one document, or can it be multiple documents? In the UI, it says "Documents".

Author: In process. Thanks.

* The bell icon allows you to open the **Create detector alert trigger** pane, where you can quickly set up an alert for the specific finding and modify rules and their conditions as required.
For details on setting up an alert, see [Set up alerts]({{site.url}}{{site.baseurl}}/security-analytics/sec-analytics-config/detectors-config/#step-3-set-up-alerts) in detector creation documentation.

> Each finding in the list also includes a **Finding ID**. You can select the ID as another option for opening the **Finding details** pane. An example of **Finding details** is shown in the following image.

Reviewer: ```suggestion
Each finding in the list also includes a **Finding ID**. You can select the ID to open the **Finding details** pane. An example of **Finding details** is shown in the following image.
```

Author: In Actions, content that precedes this, it gives you a first way of opening the Finding details pane. This would be the second. 
But if someone were to look at this without reading a few sentences above, I can see how it would seem awkward. Will reword.

<img src="{{site.url}}{{site.baseurl}}/images/Security/findings1.png" alt="Finding details pane" width="60%">

### Viewing surrounding documents

The **Finding details** pane contains specific information about the finding, including the document that generated the finding. To investigate the series of events that led to the finding or followed the finding, you can select **View surrounding documents** to open the document in the **Discover** panel and view other documents preceding or following it.

1. Open **Finding details** by selecting the **Finding ID** in the list of findings.
1. In the **Documents** section, select **View surrounding documents**. If an index pattern already exists for the document, the **Discover** panel opens and displays the document. If an index pattern does not exist, the **Create index pattern to view documents** window opens and prompts you to create an index pattern, as shown in the following image.

> <br><img src="{{site.url}}{{site.baseurl}}/images/Security/findings2.png" alt="popup window prompting users to create an index pattern" width="60%">

Reviewer: Remove `<br>` and add a new line before and after the image.

Author: These images are in steps. If I add spaces it breaks the sequence and doesn't render the step numbers correctly. Maybe you're suggesting there's a hard break that leaves the steps intact. Will look into it.
Update: looked and found. Should be all set.

Reviewer: If you need an image to be under a step, tab it over one tab. The tabbing makes the list intact.

1. In the **Create index pattern to view documents** window, the index pattern name is automatically populated. Enter the appropriate time field from the log index used to determine the timing for log events. For information on mapping log fields to detector fields, see [Step 2. Create field mappings]({{site.url}}{{site.baseurl}}/security-analytics/sec-analytics-config/detectors-config/#step-2-create-field-mappings). Select **Create index pattern**. The **Create index pattern to view documents** confirmation window opens.
1. Select **View surrounding documents** in the confirmation window. The **Discover** panel opens, as shown in the following image.

> <br><img src="{{site.url}}{{site.baseurl}}/images/Security/findings4.png" alt="popup window prompting users to create an index pattern" width="85%">

Reviewer: Remove `<br>` and add a new line before and after the image.

Reviewer: Also, change alternate text in the image.

Author: RE: <br> as above.

Good catch. Changed alternate text.

> The **Discover** panel displays the document that generated the finding with a light blue background. Other documents that came either before or after the event are also displayed.

Reviewer: It's best not to use the color when referring to UI elements for accessibility. We can say "highlighted" or "selected" instead.

Author: I like "highlighted". Good suggestion.
